scripts/generate-values/README.rst

```diff
@@ -76,13 +82,25 @@ Call the script:
 
    $ ./generate-values.py
 
+Note that issues have been observed using the virtualenv approach in a quite
```

Comment:
hmm, just did this:

```
docker run --rm -ti python:3.9-slim bash
apt update -y
apt install -y build-essential git
git clone https://github.com/swissdatasciencecenter/renku
cd renku/scripts/generate-values
pip install virtualenv
virtualenv .venv
source .venv/bin/activate
pip install -r requirements.txt
./generate-values.py --gitlab
```

and it worked fine... ? Can you reproduce the problem? Otherwise I'd say we can just say here "if you experience issues with the python script, we recommend you try the docker-based approach below." or something similar.